### DISCUSSION TITLE
Homeassistant Addon Rauchmelder als binary_sensor.

### DISCUSSION THREAD

**Discussion by meistermolli** [OpenCCU/OpenCCU#2128] [🇩🇪 Ideen]
Hallo,

**EDIT: nach weiterem suchen ist das sicher eher ein Thema für Custom Homematic(IP) Local**
https://github.com/danielperna84/hahomematic/issues/579
Thema hier beschrieben

stehe gerade vor der Übergabe eines HmIP Rauchmelders im addon Raspberrymatic
über Custom Homematic(IP) Local
         Homeassistant
    an Homekit.

Homeassistant und Homekit erwarten einen binary_sensor in dem in der Entity_ID smoke vorkommt:
binary_sensor.flur_rauchmelder_smoke....

Übergeben wird der jedoch von raspberrymatic als sensor.flur_rauchmelder_smoke_detector_alarm_status

Ist es möglich diese im Addon anders zu übergeben? Oder sollte ich hier einen Clon erzeugen vom sensor als binary_sensor.

Vielleicht kann mir ja jemand helfen. Weis nicht so recht ob ich hier falsch liege oder ob dies ein Feature Request sein könnte?

https://www.home-assistant.io/integrations/binary_sensor/
https://www.home-assistant.io/integrations/homekit/

Merci und Gruß 

meistermolli

**Comment by Baxxy13** [comment]
Wie die einzelnen Geräte in HA "dargestellt" werden ist Sache der Integration.
Von RaspberryMatic-Seite kann und muss man da nix machen.

Da der SWSD aber 4 States hat ist er eben kein Binary_Sensor.

**Comment by SukramJ** [comment]
@meistermolli 
Für die Custom Integration [Homematic(IP) Local](https://github.com/danielperna84/custom_homematic) findest du den [Issue Tracker ](https://github.com/danielperna84/hahomematic/issues)und das [Diskussionforum](https://github.com/danielperna84/hahomematic/discussions) im Repository der verwendeten Backend Lib [HAHomematic](https://github.com/danielperna84/hahomematic).
Dort findest Du auch einen Diskussionsbeitrag zum Thema [SWSD](https://github.com/danielperna84/hahomematic/discussions/759) und auch Beispiel für in Homekit verwendbare [Templates](https://github.com/danielperna84/hahomematic/discussions/759#discussioncomment-4410766), die auch im [Wiki](https://github.com/danielperna84/hahomematic/wiki/Community-Templates-for-Home-Assistant#split-hmip-swsd-smoke-detector-into-binary-sensors-by-christophcaina) abgelegt wurden.

**Comment by meistermolli** [comment]
Ja merci. Hatte ich ja bereits vor euren Kommentaren editiert. Ich kann nur die Diskussion nicht schließen.

Das templates Beispiel ist aber sehr gut danke!

kann man die Diskussion irgendwie schließen?
